cnxepub/tests/test_models.py

```diff
@@ -134,6 +134,19 @@ def test_sanitize_xml(self):
, the period of the operation of the wipers is:</p>\n<div class="unnumbered" data-label="" data-type="equation" id="fs-id1172097163675"><math display="block"><semantics><mrow><mrow><mi>t</mi><mo>=</mo><mtext>&#8722;</mtext><mi>R</mi><mi>C</mi><mspace width="0.2em"/><mtext>ln</mtext><mrow><mo>(</mo><mrow><mn>1</mn><mo>&#8722;</mo><mfrac><mrow><msub><mi>V</mi><mrow><mtext>out</mtext></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mi>V</mi></mfrac></mrow><mo>)</mo></mrow><mo>=</mo><mtext>&#8722;</mtext><mrow><mo>(</mo><mrow><mn>10</mn><mspace width="0.2em"/><mo>&#215;</mo><mspace width="0.2em"/><msup><mrow><mn>10</mn></mrow><mrow><mn>&#8722;3</mn></mrow></msup><mspace width="0.2em"/><mtext>F</mtext></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mn>10</mn><mspace width="0.2em"/><mo>&#215;</mo><mspace width="0.2em"/><msup><mrow><mn>10</mn></mrow><mn>3</mn></msup><mspace width="0.2em"/><mtext>&#937;</mtext></mrow><mo>)</mo></mrow><mtext>ln</mtext><mrow><mo>(</mo><mrow><mn>1</mn><mo>&#8722;</mo><mfrac><mrow><mn>10</mn><mspace width="0.2em"/><mtext>V</mtext></mrow><mrow><mn>12</mn><mspace width="0.2em"/><mtext>V</mtext></mrow></mfrac></mrow><mo>)</mo></mrow><mo>=</mo><mn>179.18</mn><mspace width="0.2em"/><mtext>s</mtext><mo>=</mo><mn>2.98</mn><mspace width="0.2em"/><mtext>min</mtext><mo>.</mo></mrow></mrow><annotation-xml encoding="MathML-Content"><mrow><mi>t</mi><mo>=</mo><mtext>&#8722;</mtext><mi>R</mi><mi>C</mi><mspace width="0.2em"/><mtext>ln</mtext><mrow><mo>(</mo><mrow><mn>1</mn><mo>&#8722;</mo><mfrac><mrow><msub><mi>V</mi><mrow><mtext>out</mtext></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mi>V</mi></mfrac></mrow><mo>)</mo></mrow><mo>=</mo><mtext>&#8722;</mtext><mrow><mo>(</mo><mrow><mn>10</mn><mspace width="0.2em"/><mo>&#215;</mo><mspace width="0.2em"/><msup><mrow><mn>10</mn></mrow><mrow><mn>&#8722;3</mn></mrow></msup><mspace width="0.2em"/><mtext>F</mtext></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mn>10</mn><mspace width="0.2em"/><mo>&#215;</mo><mspace width="0.2em"/><msup><mrow><mn>10</mn></mrow><mn>3</mn></msup><mspace width="0.2em"/><mtext>&#937;</mtext></mrow><mo>)</mo></mrow><mtext>ln</mtext><mrow><mo>(</mo><mrow><mn>1</mn><mo>&#8722;</mo><mfrac><mrow><mn>10</mn><mspace width="0.2em"/><mtext>V</mtext></mrow><mrow><mn>12</mn><mspace width="0.2em"/><mtext>V</mtext></mrow></mfrac></mrow><mo>)</mo></mrow><mo>=</mo><mn>179.18</mn><mspace width="0.2em"/><mtext>s</mtext><mo>=</mo><mn>2.98</mn><mspace width="0.2em"/><mtext>min</mtext><mo>.</mo></mrow></annotation-xml></semantics></math></div>\n</div>\n<p id="fs-id1172099623126">The <em data-effect="italics">RC</em> circuit has thousands of uses and is a very important circuit to study. Not only can it be used to time circuits, it can also be used to filter out unwanted frequencies in a circuit and used in power supplies, like the one for your computer, to help turn ac voltage to dc voltage.</p>\n</section>\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n  \n  \n  </div>"""
```

Comment:
These really long lines cause Travis to fail (the linter complains that they are too long)